less/mosaico/static-pages/abi.less

```diff
@@ -0,0 +1,16 @@
+.static-page__abi {
+  padding: 60px 0;
+  .abi__headline {
```

Comment:
If font-size is bigger than 1.5rem/24px, then some of the words in the headline are too long to fit on one line, so that might need some rules.